ci-operator/jobs/openshift/cluster-network-operator/openshift-cluster-network-operator-release-4.3-presubmits.yaml

```diff
@@ -65,71 +65,6 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )e2e-gcp,?($|\s.*)
-  - agent: kubernetes
-    always_run: false
-    branches:
-    - release-4.3
-    cluster: api.ci
-    context: ci/prow/e2e-gcp-ovn-upgrade
-    decorate: true
-    decoration_config:
-      skip_cloning: true
-    labels:
-      ci-operator.openshift.io/prowgen-controlled: "true"
-      pj-rehearse.openshift.io/can-be-rehearsed: "true"
-    name: pull-ci-openshift-cluster-network-operator-release-4.3-e2e-gcp-ovn-upgrade
```

Comment:
you deleted all the non-`master` copies of the jobs

Reply:
@danwinship I did, because we don't care about OVN upgrades on anything lower really... I guess I can bring them back.

Reply:
oh, sure, the 4.3-4.5 stuff can go but you should leave the 4.6 and 4.7 versions

Reply:
er, well, and probably 4.5 too. The upgrade jobs are not testing major version upgrades, they're testing minor version upgrades, which we definitely care about in any release where any customer is running ovn-kubernetes